websites/D/Discord.style/presence.ts

```diff
@@ -4,12 +4,28 @@ const presence = new Presence({
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
 presence.on("UpdateData", async () => {
+	let presenceData: PresenceData = {
+		largeImageKey: "discordstyle_logo",
+	};
+
 	const showTimestamp = await presence.getSetting<boolean>("timestamp"),
 		showButtons = await presence.getSetting<boolean>("buttons"),
-		presenceData: PresenceData = {
-			largeImageKey: "discordstyle_logo",
+		pages: Record<string, PresenceData> = {
+			"/bot": {
+				details: "Viewing 🤖 bot page",
+				buttons: [{ label: "View Page", url: document.location.href }],
+			},
+			"/search": {
+				details: "Viewing search page",
+				buttons: [{ label: "View Page", url: document.location.href }],
```

Comment:
`document.location` properties are accessed many times. For consistency, they should be destructured.